### DISCUSSION TITLE
期望有 组合和解除组合 的功能

### DISCUSSION THREAD

**Discussion by GreatAuk** [antvis/XFlow#58] [Q&A] [answered]
![](https://user-images.githubusercontent.com/42988574/140466626-b0997c2a-e314-4880-b3a0-f1f1afd8683c.gif)

**Comment by lyn-boyu** [comment, marked as answer]
在DAG/流程图解决方案中有群组的方式，https://xflow.antv.vision/docs/tutorial/solutions/dag
